docs/sources/features/datasources/azuremonitor.md

```diff
@@ -222,25 +238,27 @@ AzureActivity
 | order by TimeGenerated desc
 ```
 
+If your credentials give you access to multiple subscriptions then choose the appropriate subscription first.
+
 {{< docs-imagebox img="/img/docs/v60/azureloganalytics-service-query-editor.png" class="docs-image--no-shadow" caption="Azure Log Analytics Query Editor" >}}
 
 ### Azure Log Analytics Macros
 
 To make writing queries easier there are several Grafana macros that can be used in the where clause of a query:
 
 - `$__timeFilter()` - Expands to
-    `TimeGenerated ≥ datetime(2018-06-05T18:09:58.907Z) and`
-    `TimeGenerated ≤ datetime(2018-06-05T20:09:58.907Z)` where the from and to datetimes are from the Grafana time picker.
+  `TimeGenerated ≥ datetime(2018-06-05T18:09:58.907Z) and`
+  `TimeGenerated ≤ datetime(2018-06-05T20:09:58.907Z)` where the from and to datetimes are from the Grafana time picker.
 
 - `$__timeFilter(datetimeColumn)` - Expands to
-    `datetimeColumn  ≥ datetime(2018-06-05T18:09:58.907Z) and`
-    `datetimeColumn ≤ datetime(2018-06-05T20:09:58.907Z)` where the from and to datetimes are from the Grafana time picker.
+  `datetimeColumn ≥ datetime(2018-06-05T18:09:58.907Z) and`
+  `datetimeColumn ≤ datetime(2018-06-05T20:09:58.907Z)` where the from and to datetimes are from the Grafana time picker.
 
-- `$__escapeMulti($myVar)` - is to be used with multi-value template variables that contains illegal characters. If $myVar has the value  `'\\grafana-vm\Network(eth0)\Total','\\hello!'`, it expands to: `@'\\grafana-vm\Network(eth0)\Total', @'\\hello!'`. If using single value variables there no need for this macro, simply escape the variable inline instead - `@'\$myVar'`
+- `$__escapeMulti($myVar)` - is to be used with multi-value template variables that contains illegal characters. If \$myVar has the value `'\\grafana-vm\Network(eth0)\Total','\\hello!'`, it expands to: `@'\\grafana-vm\Network(eth0)\Total', @'\\hello!'`. If using single value variables there no need for this macro, simply escape the variable inline instead - `@'\$myVar'`
```

Comment:
Should `\$myVar` really render as `\$myVar` in all occurrences? 

Looks a bit weird on staging to me: http://staging-docs.grafana.org/features/datasources/azuremonitor/#azure-log-analytics-macros

Reply:
No - nothing to do with this change but can fix.